CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,13 @@
 == 3030-12-31 Getting Started
 <p>If you're new to Constellation, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.gettingstarted">getting started guide</a>.</p>
 
+== 2025-07-23 Add spell checking in Notes View
+<p>Spell check now added to following functions:
+<p>Notes View -> New Note Pane
+<p>Conversation View -> Create Translation / Edit Translation
+<p>Import/Export -> New Attribute Dialog
+<p>Attribute Editor -> All string attributes (Spell checking in Attribute Editor is default to turned off. User can right click to 'Turn on Spell checking')
+
```

Comment:
Would it be better to say `Spell checking in Attribute Editor is turned off by default.`?

Reply:
done